src/new-client/src/app/lib/services/appointmentsService.ts

```diff
@@ -181,7 +184,7 @@ export const saveUserRoleAssignments = async (
 
 export const saveSiteAttributeValues = async (
   site: string,
-  attributeValues: AttributeValue[],
+  attributeValues: SetAttributes,
```

Comment:
Very minor thing, but just for consistency, elsewhere we've explicitly named request/response types (i.e. `ApplyAvailabilityTemplateRequest`. Could we make this `SetAttributesRequest` or something similar?